python/py-flask/Portfile

```diff
@@ -47,14 +47,9 @@ if {${name} ne ${subport}} {
         depends_lib-append  port:py${python.version}-importlib-metadata
     }
 
-    depends_test-append     port:py${python.version}-pytest
-
     # test before 2.x requires missed py-pathlib
     if {${python.version} > 27} {
```

Comment:
Should tests be enabled across-the-board, such that the version check can be removed? And can the comment now be removed, since it's no longer relevant?

Reply:
@mascguy `py-pathlib` is still missed (MacPorts has `py-pathlib2`) and tests aren't working on python2.7 :) => seems that comment is relevant.

Reply:
I may add py-pathlib to enable all tests here but it seems strange because the last release of this library was 9 years ago...